### PR TITLE
[IIIF-915] Add Unit Test for CheckEndpointsHandler

### PR DESCRIPTION
**Pull request by DRickard**
Updates:
- Added CheckEndpointsHandlerTest.java
- Added debug message to fester_messages.xml

### CODE REVIEW

File: src/test/java/edu/ucla/library/iiif/fester/handlers/CheckEndpointsHandlerTest.java [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+
+        webClient
+            .get(port, Constants.UNSPECIFIED_HOST, requestPath)
+            .expect(ResponsePredicate.SC_SUCCESS)
+            .as(BodyCodec.jsonObject())
+            .send(ar -> {
+                final int statusCode = ar.result().statusCode();
+
+                if (ar.succeeded()) {
+                    final JsonObject aResult = ar.result().body();
```
Comment (ksclarke) [inline src/test/java/edu/ucla/library/iiif/fester/handlers/CheckEndpointsHandlerTest.java:53]: Oops, just noticed the aResult... we'd like to reserve the a* names for parameters that come in through a method so we can determine where they're defined.

File: src/test/java/edu/ucla/library/iiif/fester/handlers/CheckEndpointsHandlerTest.java [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+    private static final Logger LOGGER = LoggerFactory.getLogger(CheckEndpointsHandlerTest.class, Constants.MESSAGES);
+
+    /**
+     * Test the CheckEndpointsHandler
+     *
+     * @param aContext A testing context
+     */
+    @Test
+    public void testCheckEndpointsHandler(final TestContext aContext) {
+        final String expectedStatus = Status.OK;
```
Comment (ksclarke) [inline src/test/java/edu/ucla/library/iiif/fester/handlers/CheckEndpointsHandlerTest.java:37]: I guess also... what's the value to setting expectedStatus to the constant? Why not just use the constant where expectedStatus is used?

File: src/main/resources/fester_messages.xml [hunk 3]
```diff
@@ -163,4 +163,5 @@
   <entry key="MFS-147">Festerize is outdated, please upgrade to v{} (see README for instructions)</entry>
   <entry key="MFS-148">S3 endpoints healthcheck failed: {})</entry>
   <entry key="MFS-149">S3 endpoint response issue: {})</entry>
+  <entry key="MFS-150">Test CheckEndpointsHandler with request path: {})</entry>
 </properties>
```
Comment (markmatney) [inline src/main/resources/fester_messages.xml:167]: these last few messages have unmatched closing parens

File: src/test/java/edu/ucla/library/iiif/fester/handlers/CheckEndpointsHandlerTest.java (hunk 1 above)
Comment (DRickard) [reply src/test/java/edu/ucla/library/iiif/fester/handlers/CheckEndpointsHandlerTest.java]: Changed variable name

File: src/test/java/edu/ucla/library/iiif/fester/handlers/CheckEndpointsHandlerTest.java (hunk 2 above)
Comment (DRickard) [reply src/test/java/edu/ucla/library/iiif/fester/handlers/CheckEndpointsHandlerTest.java]: Removed unneeded variable.

File: src/main/resources/fester_messages.xml (hunk 3 above)
Comment (DRickard) [reply src/main/resources/fester_messages.xml:167]: Fixed the typos

**Pull request merged**